src/workerd/server/server.c++

```diff
@@ -1341,8 +1343,13 @@ public:
 
   class ActorNamespace final: private kj::TaskSet::ErrorHandler {
   public:
-    ActorNamespace(WorkerService& service, kj::StringPtr className, const ActorConfig& config)
-        : service(service), className(className), config(config), onBrokenTasks(*this) {}
+    ActorNamespace(WorkerService& service,kj::StringPtr className, const ActorConfig& config,
+        kj::Timer& timer)
+        : service(service),
+          className(className),
+          config(config),
+          onBrokenTasks(kj::HashMap<kj::String, kj::Maybe<kj::Promise<void>>>()),
```

Comment:
You can delete this line since it's ultimately resulting in default-initialization.